test/go2_test.cpp

```diff
@@ -20,6 +20,9 @@
 #include <sstream>
 #include <string>
 
+const bool STORE_PREDICTIONS = true; // If true the serow base estimates are stored under results/
```

Comment:
```suggestion
constexpr bool kStorePredictions = true; // If true the serow base estimates are stored under results/
```
https://google.github.io/styleguide/cppguide.html#Constant_Names